Wabbajack/View Models/MainWindowVM.cs

```diff
@@ -144,7 +148,15 @@ public MainWindowVM(MainWindow mainWindow, MainSettings settings)
 
             OpenTerminalCommand = ReactiveCommand.Create(() => OpenTerminal());
             
-
+            _percentCompleted = Observable.CombineLatest(
+                this.WhenAny(x => x.Installer.Value.PercentCompleted),
+                this.WhenAny(x => x.Compiler.Value.PercentCompleted),
+                (Percent installer, Percent compiler) =>
+                {
+                    // add logic to take te active percentage, installer or compiler
+                    return installer;
```

Comment:
You didn't sub in any logic for selecting the proper percent to use.  Right now it's always going to show the installer's percent, and will never make use of the compiler's percent.

You might be wondering how to tell which to choose/use:
 - Can either check and return whichever one is not 0 or 1, as a hacky but probably decently accurate way.
 - Can add WhenAny calls and retrieve CompilerVM's `Compiling` property, and the similar one from InstallerVM.  By adding them into the properties that will trigger a refresh, you'll also need to add them as parameters to the lambda for use.  Now the lambda knows which side is the active one, so it'll know which percentage to return for sure.